portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -145,13 +145,54 @@ definition {
 		}
 	}
 
-	@description = "Feature ID: Legacy | Automation ID: LRAC-9113 | Test Summary: It is possible to create multiple properties in different workspaces"
-	@ignore = "true"
+	@description = "Bug: 7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"
```

Comment:
Hey @manuferreira, you forgot LRAC in the bug ticket:

`@description = "Bug: LRAC-7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"`